examples/vnet/doca_gw_dpdk.c

```diff
@@ -68,18 +99,190 @@ int doca_gw_port_stop(struct doca_gw_port *port)
     }
     return 0;
 }
+static void build_eth_flow_item(struct gw_item_param  *param,
+					struct doca_gw_match *match)
+{
+	struct rte_flow_item *item = param->item;
+
+	match = match;
+	item->type = RTE_FLOW_ITEM_TYPE_ETH;
+}
+
+#define GET_SRC_IP(match, type) ((type == OUTER_IP) ? match->out_src_ip : match->in_src_ip)
```

Comment:
please put macros on top of the file also use proper name DOCA_GW..... in this case also IPV4